packages/iov-tendermint-rpc/src/v0-25/requests.ts

```diff
@@ -96,12 +96,13 @@ interface RpcAbciQueryParams {
   readonly height?: string;
   readonly trusted?: boolean;
 }
+
 function encodeAbciQueryParams(params: requests.AbciQueryParams): RpcAbciQueryParams {
   return {
     path: notEmpty(params.path),
     data: Encoding.toHex(params.data) as HexString,
     height: may(Integer.encode, params.height),
-    trusted: params.trusted,
+    trusted: params.prove,
```

Comment:
`trusted: !params.prove`
they are opposites

Reply:
oh! oh oh! well spotted

Reply:
Like this? https://github.com/iov-one/iov-core/pull/725/commits/8b5771288f65bef022799f87c91d6ca235c17fa8

Reply:
Yes, that looks like a good fix